src/Spain/Mews.Fiscalizations.Spain/Model/Response/ErrorResult.cs

```diff
@@ -19,10 +20,11 @@ private ErrorResult(string code, string message)
 
         public static ErrorResult Create(string code, string message) => new ErrorResult(code, message);
 
-        private ErrorType ConvertErrorType(string message)
+        private ErrorType MapErrorType(string message)
         {
             return message.Match(
-                "Codigo[401].Certificado revocado", _ => ErrorType.CertificateRevoked);
+                "Codigo[401].Certificado revocado", _ => ErrorType.CertificateRevoked,
+                _ => ErrorType.Unknown);
```

Comment:
Move the closing parentheses to the next line and align it with `return message.Match(`